### DISCUSSION TITLE
Katex/Latex support?

### DISCUSSION THREAD

**Discussion by FossPrime** [markdoc/markdoc#15] [Q&A] [answered]
I didn't see anything about how to use plugins... Or what spec exactly is supported, GFM I presume? If so is there a way to add Katex or Latex?

**Comment by mfix-stripe** [comment, marked as answer]
Markdoc follows the [CommonMark specification](https://commonmark.org/) (as outlined in our docs here: https://markdoc.io/docs/faq). For that reason, Katex and Latex aren't supported out of the box, but you can create a custom tag (https://markdoc.io/docs/tags) that parses the Latex content in the `transform` function. 

If you end up creating this tag, please share it here!

**Reply by FossPrime** [reply]
Thank you. I really need GFM style table support as we use GitLab's excellent GUI for Markdown content editing and it, like many, often prefers the GitHub Flavoured Markdown style of tables vs the html style.

https://github.github.com/gfm/

However, I really like the list style tables of Markdoc and may adopt them if I can convince GitLab to use them or if I find a better GUI that supports them. Drag and drop image support was GitLab's content editor's killer feature for us. We throw the uploads folder on S3 and bobs our uncle.

My apologies for not reading the FAQ, it wasnt displaying on my phone due to bug #16

**Reply by FossPrime** [reply]
The tricky part is supporting this securely... Katex renderers are notoriously insecure... that's where the recently open sourced V8 powered, Lambda alternative with sub millisecond boot times, CloudFlare Workers might be a perfect fit.
